optics/optics.cabal

```diff
@@ -132,6 +132,10 @@ test-suite optics-tests
   if impl(ghcjs)
     buildable: False
 
+  -- Doesn't compile because of inspection-testing
```

Comment:
There is patch in [head.hackage](https://gitlab.haskell.org/ghc/head.hackage), https://gitlab.haskell.org/ghc/head.hackage/-/blob/master/patches/inspection-testing-0.4.2.4.patch,

doesn't it work?